src/components/avatar/AvatarInitials.js

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box, { pickBoxProps } from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+import { Heading4 } from '../..';
+
+class AvatarInitials extends PureComponent {
+  getInitials = () => {
+    const { name } = this.props;
+    const nameSplitted = name.split(' ');
+    const firstLetter = nameSplitted[0].charAt(0);
+    if (nameSplitted.length > 1) {
+      const lastLetter = nameSplitted[nameSplitted.length - 1].charAt(0);
```

Comment:
It looks like it's not possible to get 3 initials or more, so I'd just go for `nameSplitted[1].charAt(0)`

Reply:
We always want the first letter of the first and last name, so if the name would be: "Peter Maxim Karl Weytens" the result needs to be: "PW". Otherwise it would be "PM".

Reply:
Aha fair enough, you're right 👍